api_docs/message-formatting.md

```diff
@@ -23,6 +23,12 @@ for syntax highlighting. This field is used in the
 mentions][help-global-time] to supported Markdown message formatting
 features.
 
+## Mentions
+
+**Changes**: In Zulip 9.0 (feature level 247), `channel` was added
+to the supported [wildcard][help-mention-all] options used in the
+[mentions][help-mentions] Markdown message formatting feature.
```

Comment:
@amanagr @roanster007 so now that we have this new system for documenting API changes to ... did we forget to note in the API documentation when that changed? We can add that here, and maybe #27007 should be updated to use this mechanism too.